### ISSUE TITLE
NZBGet Deleted/Copy History events are treated as failed, but aren't tracked so they aren't redownloaded

### ISSUE THREAD

**Issue by markus101** [Sonarr/Sonarr#1437]
This log snippet shows that a drone parameter was set, but Sonarr sees it as failed, but not grabbed, I suspect NZBGet is telling Sonarr it failed to add and it's never added to Sonarr's history. If that's the case we should try to determine if it failed for this reason and add it to history so it can be dealt with by failed download handling or immediately treat it as failed.

```
{
"ID" : 2528,
"Name" : "Celebrity.Family.Feud.2015.S02E08.720p.HDTV.x264-W4F",
"RemainingFileCount" : 0,
"HistoryTime" : 1471977365,
"Status" : "DELETED/COPY",
"Log" : [],
"NZBID" : 2528,
"NZBName" : "Celebrity.Family.Feud.2015.S02E08.720p.HDTV.x264-W4F",
"NZBNicename" : "Celebrity.Family.Feud.2015.S02E08.720p.HDTV.x264-W4F",
"Kind" : "NZB",
"URL" : "",
"NZBFilename" : "Celebrity.Family.Feud.2015.S02E08.720p.HDTV.x264-W4F.nzb",
"DestDir" : "G:\\incomplete\\Celebrity.Family.Feud.2015.S02E08.720p.HDTV.x264-W4F.#2528",
"FinalDir" : "",
"Category" : "tv",
"ParStatus" : "NONE",
"ExParStatus" : "NONE",
"UnpackStatus" : "NONE",
"MoveStatus" : "NONE",
"ScriptStatus" : "NONE",
"DeleteStatus" : "COPY",
"MarkStatus" : "NONE",
"UrlStatus" : "NONE",
"FileSizeLo" : 1935273098,
"FileSizeHi" : 0,
"FileSizeMB" : 1845,
"FileCount" : 54,
"MinPostTime" : 1471976247,
"MaxPostTime" : 1471976322,
"TotalArticles" : 2506,
"SuccessArticles" : 0,
"FailedArticles" : 0,
"Health" : 1000,
"CriticalHealth" : 889,
"DupeKey" : "",
"DupeScore" : 0,
"DupeMode" : "ALL",
"Deleted" : true,
"DownloadedSizeLo" : 0,
"DownloadedSizeHi" : 0,
"DownloadedSizeMB" : 0,
"DownloadTimeSec" : 0,
"PostTotalTimeSec" : 0,
"ParTimeSec" : 0,
"RepairTimeSec" : 0,
"UnpackTimeSec" : 0,
"MessageCount" : 4,
"ExtraParBlocks" : 0,
"Parameters" : [
{
"Name" : "drone",
"Value" : "d51891f8bdbc4481a3f0a213d8d41c3b"
},
{
"Name" : "*Unpack:",
"Value" : "yes"
},
{
"Name" : "FakeDetector.py:",
"Value" : "yes"
},
{
"Name" : "DeleteSamples.py:",
"Value" : "yes"
}],
"ScriptStatuses" : [
],
"ServerStats" : [
]
},
```

**Comment by markus101** [comment]
Related nzbget issue:

https://github.com/nzbget/nzbget/issues/284

**Comment by thezoggy** [comment]
looks like its fixed in      v18.0-r1820  ?

**Comment by markus101** [comment]
This was fixed in: https://github.com/Sonarr/Sonarr/commit/072ca459bd68ae2e7d8d8f45eb22676dd36e0007
